build-test.cmd

```diff
@@ -265,7 +269,38 @@ if errorlevel 1 (
     exit /b 1
 )
 
+REM Prepare the Test Drop
+REM Cleans any NI from the last run
+powershell "Get-ChildItem -path %__TestWorkingDir% -Include '*.ni.*' -Recurse -Force | Remove-Item -force"
```

Comment:
Yes that is not great, but at least it solves the issue [ Actually that was how it used to be, the core_root produced by builtest was used by xplat]

The other way is to make buildtest call runtest to create core_root.
I am fine with both approaches